assets/js/payplug-apple-pay.js

```diff
@@ -121,6 +122,24 @@
 					},
 				})
 			}
+		},
+
+		getOrderTotals(){
+			if(!apple_pay.load_order_total){
```

Comment:
this is for the first time the pages load to not make the call right?
Can you put a little comment with this info?